src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService regServ;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        regServ = new RegistrationServiceImpl();
+        user = new User();
```

Comment:
move `user = new User();` to `BeforeEach` method

Reply:
ok. It was like this in my code from begining...